lib/administrate/generator_helpers.rb

```diff
@@ -4,10 +4,29 @@ def call_generator(generator, *args)
       Rails::Generators.invoke(generator, args, generator_options)
     end
 
+    def find_routes_file
+      routes_file = Rails.root.join("config/routes.rb")
+      if !File.file?(routes_file)
```

Comment:
There's a `exist?` method on `File` which might avoid us flipping with `!`. What do you think?

Reply:
Sounds great, I'll update that.